qaframework-bdd-tests/cypress/integration/cucumber/step_definitions/oclsubscriptionmodule/ocl-subscription-module.js

```diff
@@ -25,15 +18,15 @@ Given('the user goes to the openmrs application', () => {
 });
 
 When('the user logs into the Dictionary Manager', () => {
-  cy.get("#username").type(OCL_USER_NAME);
-  cy.get("#password").type(OCL_USER_PASSWORD);
+  cy.get("#username").type("openmrs");
+  cy.get("#password").type("Openmrs123");
   cy.get("#login-form form").submit();
   cy.url().should('contain', 'https://openmrs.staging.openconceptlab.org/user/collections/');
 });
 
 When('the user logs into the openmrs Ref App', () => {
-  cy.get("#username").type(OPENMRS_USER_NAME);
-  cy.get("#password").type(OPENMRS_USER_PASSWORD);
+  cy.get("#username").type("admin");
```

Comment:
@kdaud   why this change. what happens when its left the way it was initially.

Reply:
The server credentials fails to be exported hence throwing Cypress error: `cy.type() can not accept an empty string`, on the the other hand as suggested in the PR when the credentials are passed directly as strings the test passes.

Reply:
> `cy.type() can not accept an empty string`,

You mean even when you  equate that string  to a value, something like String` USER_NAME  = "Admin"`, it still fails ?

Reply:
The credentials are set in command.js file
```js
export const OCL_USER_NAME = Cypress.env('OCL_USER_NAME');
export const OCL_USER_PASSWORD = Cypress.env('OCL_USER_PASSWORD');
export const OPENMRS_USER_NAME = Cypress.env('OPENMRS_USER_NAME');
export const OPENMRS_USER_PASSWORD = Cypress.env('OPENMRS_USER_PASSWORD');
export const OCL_USER_TOKEN = Cypress.env('OCL_USER_TOKEN'
```
which are then imported as 

```js
import { 
  OCL_USER_NAME, 
  OCL_USER_PASSWORD, 
  OCL_USER_TOKEN,
  OPENMRS_USER_NAME, 
  OPENMRS_USER_PASSWORD
} from '../../../../support/commands';
```
Then passed in their respective fields. However they are all failing to be imported hence throwing the error I shared above.

Reply:
Are they exported, Usually react js, js classes need to be exported first for them to be accessed.

Reply:
@kdaud  Seems even the export function does not work as expected. Look here https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/cypress/support/commands.js#L10

Reply:
Do you have a suggestion that can use rather?

Reply:
If this really failed with that initial work, then we can go with direct placing of those attribute.

Reply:
Have not gotten a way of importing them after trying several alternatives I know. If there is one you know then we can use that rather otherwise we can have them passed directly as strings and if we get a way out  then we can do refactor the test.

Reply:
> we can have them passed directly as strings and if we get a way out then we can do refactor the test.

I agree because i have gone through it microscopically and am conviced this looks good.